lang/cs/.nuget/NuGet.targets

```diff
@@ -128,13 +128,17 @@ under the License.
         <DownloadNuGet OutputFilename="$(NuGetExePath)" Condition=" '$(DownloadNuGetExe)' == 'true' AND !Exists('$(NuGetExePath)')" />
     </Target>
 
-    <Target Name="RestorePackages" DependsOnTargets="CheckPrerequisites">        
+    <Target Name="RestorePackages" DependsOnTargets="CheckPrerequisites">
         <Exec Command="$(RestoreCommand)"
               Condition="'$(OS)' != 'Windows_NT' And Exists('$(PackagesConfig)')" />
 
         <Exec Command="$(RestoreCommand)"
               LogStandardErrorAsError="true"
               Condition="'$(OS)' == 'Windows_NT' And Exists('$(PackagesConfig)')" />
+
+        <!-- Restore solution-level packages as well -->
+        <Exec Command="$(NuGetCommand) restore $(SolutionDir)\.nuget\packages.config -o $(PackagesDir)"
```

Comment:
This is the same behavior as we currently have without my change - if we delete packages folder, opening VS solution will not restore packages, only first build attempt will restore them. I don't think there's a way to call restore on "open in VS" action.